SalesforceCommerceCloud/pwa-kit
# @W-16544327 social login UI

This adds a SocialLogin component that is a stack of buttons for social login configured through default.js. The actions aren't in the SDK yet and Jinsu is working on passwordless login that will contain this new component.


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Add social login component
- Add icons for components
- Add optional key to default.js
- Add tests

# How to Test-Drive This PR

- Start locally and check that the login screen is normal
- Add the following to `default.js` under the `app` key
```
        login: {
            idps: ['Apple', 'Google']
        },
```
- Restart locally and see the new buttons for login

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
